### PR TITLE
Implement work around for text clipping issue.

### LINKED ISSUE #1773: TeachingTip Icon is clipped
**Describe the bug**

TeachingTip icon is clipped when the Tip is opened for the second time. (See screenshots below)

**Steps to reproduce the bug**

1. Launch Xaml Controls Gallery
2. Navigate to TeachingTip page
3. Click the "Show TeachingTip" button. Notice that the tip displays correctly
4. Close the Teaching tip
5. Repeat step 3 and reopen the TeachingTip. Notice that the refresh icon is clipped. 

**Expected behavior**

Icon is not clipped.

**Screenshots**

image

**Version Info**

Version	10.0.19041 Build 19041
10.0.19041

NuGet package version: 

2.3.191211002

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |  Yes   |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 | Yes |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

**Additional context**

Possibly related to #1372?

### PR DESCRIPTION
**Pull request by StephenLPeters**
fixes #1773 #1372.  This change implements a work around for the [internal issue](https://microsoft.visualstudio.com/OS/_workitems/edit/26001712).

### CODE REVIEW

File: dev/TeachingTip/TeachingTip.cpp [hunk 1]
```diff
@@ -1552,7 +1552,8 @@ void TeachingTip::CreateContractAnimation()
     m_contractElevationAnimation.set([this, compositor, contractEasingFunction]()
     {
         auto const contractElevationAnimation = compositor.CreateVector3KeyFrameAnimation();
-        contractElevationAnimation.InsertExpressionKeyFrame(1.0f, L"Vector3(this.Target.Translation.X, this.Target.Translation.Y, 0.0f)", contractEasingFunction);
+        // animating to 0.01f instead of 0.0f as work around to internal issue 26001712 which was causing text clipping.
+        contractElevationAnimation.InsertExpressionKeyFrame(1.0f, L"Vector3(this.Target.Translation.X, this.Target.Translation.Y, 0.01f)", contractEasingFunction);
```
Comment (chingucoding) [inline dev/TeachingTip/TeachingTip.cpp:1556]: Out of curiosity, do we know why this "fixes" the bug? 😅

Comment (StephenLPeters) [reply dev/TeachingTip/TeachingTip.cpp:1556]: It has to do with an over optimization that's happening in comp. The team is still looking into how we are getting into the bad state but its something around the transition from a 2d object to a 3d object.

Comment (chingucoding) [reply dev/TeachingTip/TeachingTip.cpp:1556]: Oh okay, thanks for the info!

**Pull request merged**